### ISSUE TITLE
Return versus process.exit(ERROR)

### ISSUE THREAD

**Issue by christineiym** [comp426-2022-spring/a01#11]
Hi! I am not sure if this is intended or not, but I found that if I used process.exit(1) before or instead of returning (followed by a seemingly unreached process.exit(1)), the autograder gives the following output:
<img width="583" alt="Screen Shot 2022-01-27 at 8 07 04 PM" src="https://user-images.githubusercontent.com/69741280/151469423-741c0015-0729-42f2-8c50-c0537a454365.png">

The error logged to the terminal is the same in both cases:
![image](https://user-images.githubusercontent.com/69741280/151469779-6453b070-2caf-4364-b1f6-2791025800cf.png)

Is there a reason why process.exit(1) should not come first?
Thank you!

**Comment by jdmar3** [comment]
So, in the demo, I put `process.exit(1)` in entirely the wrong place. It should actually go at the end of the entire file.

Why is this?

Because if the script gets to the end of the file on its own, then something has gone wrong and it should exit with an error. That is because the script is supposed to listen, which means it has to keep running.

HOWEVER, when we do this the right way, then the autograder catches an error, which makes the test fail EVEN WHEN the output is correct, because all it cares about at that point is the error.

The problem is that I am intentionally trying to create an error to see if you are doing error handling. 🤦🏻 

So, basically I have to configure the autograder to look for something else instead of the error itself.

**Comment by jdmar3** [comment]
Ultimately I have adjusted the assignment instructions to say not to worry about including `process.exit(1)` since really the error handling is what we care about in this assignment.
